in_toto/models/layout.py

```diff
@@ -140,8 +140,11 @@ def import_step_metadata_from_files_as_dict(self):
     """
     step_link_dict = {}
     for step in self.steps:
-      link = models__link.Link.read_from_file(step.name + '.link')
-      step_link_dict[step.name] = link
+      key_link_dict = {}
+      for keyid in step.pubkeys:
+        link = models__link.Link.read_from_file(step.name + '.' + keyid + '.link')
```

Comment:
Is there any way to change this to get only 6 bytes of just the keyid?
FILENAME_FORMAT.format(step_name=self.step_name, keyid=self.key["keyid"]))
This gets the entire keyid!

Or can we make changes to FILENAME_FORMAT = "{step_name}.{keyid}.link" to enable only the first 6 bytes?